druid/src/widget/label.rs

```diff
@@ -520,9 +522,33 @@ impl<T: TextStorage> Widget<T> for RawLabel<T> {
     #[instrument(
         name = "RawLabel",
         level = "trace",
-        skip(self, _ctx, _event, _data, _env)
+        skip(self, ctx, event, _data, _env)
     )]
-    fn event(&mut self, _ctx: &mut EventCtx, _event: &Event, _data: &mut T, _env: &Env) {}
+    fn event(&mut self, ctx: &mut EventCtx, event: &Event, _data: &mut T, _env: &Env) {
+        match event {
+            Event::MouseUp(event) => {
+                // Account for the padding
+                let mut pos = event.pos;
+                pos.x -= LABEL_X_PADDING;
```

Comment:
just a thought, but I might just write this as,

```suggestion
                let pos = event.pos - Vec2::new(LABEL_X_PADDING, 0.0);
```